CMakeLists.txt

```diff
@@ -185,7 +185,7 @@ if(NOT HAVE_CXX11_ATOMIC)
     " ${LIBATOMIC_LINK_FLAGS}")
 endif()
 
-option(WITH_RDMA "Enable RDMA in async messenger" OFF)
+option(WITH_RDMA "Enable RDMA in async messenger" ON)
```

Comment:
why do we need to enable `WITH_RDMA` by default?

Reply:
WITH_RDMA has always been enabled by default. 
Then I wanted to enable rdma by -DWITH_RDMA=ON when compiling ceph, but when spdk is not enabled, ceph will fail to compile,rdma should be enabled by default, so I changed it back.